src/gettingstarted/cli.md

```diff
@@ -97,3 +97,30 @@ Once Bash for Windows is installed, you can install the Platform.sh CLI with the
 ```bash
 curl -sS https://platform.sh/cli/installer | php
 ```
+
+## API Tokens
+
+The CLI can also be used from 3rd party CI services or other automation tools, and supports an API Token authentication option for that purpose.
```

Comment:
The "3rd party" bit is not important (it can also be used with 1st-party automation tools, as demonstrated above).

Reply:
Removed.